src/routes/GettingStartedPage/Wizard/InitialSetup.js

```diff
@@ -53,6 +53,7 @@ const InitialSetup = () => {
               className="pf-u-my-md"
               component={(props) => <AppLink {...props} to={'instances'} />}
               aria-label="Create Instance"
+              target="_blank"
```

Comment:
It may not matter in a controlled environment like this, but it's a best practice to include these safety attributes when we use target=blank.

https://help.ahrefs.com/en/articles/4684931-noreferrer-noopener-nofollow-attributes 

```suggestion
              rel="noopener noreferrer nofollow"
              target="_blank"
```